src/style.css

```diff
@@ -1 +1,82 @@
-/* Styles go here */
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  /* position: absolute; */
+  /* width: 1200px; */
+  max-width: 1200px;
+  /* min-width: 1024px; */
+  height: 60px;
+}
+
+img {
+  display: flex;
+  margin: 10px 0 10px 50px;
+  cursor: pointer;
+}
+
+.nav {
+  display: flex;
+  /* justify-content: flex-end; */
+  /* margin: 0 20px; */
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+}
+
+.nav__item {
+
```

Comment:
redundant line

Reply:
fix it https://github.com/mate-academy/layout_moyo-header/pull/2442/files#r1053638136